tools/relay/proto/helloworld/helloworld.proto

```diff
@@ -0,0 +1,38 @@
+// MIT License
```

Comment:
If I don't add the generated files then linters throw error (and `go mod tidy` wasn't working). I will check if this can be resolved without generating the files.

Reply:
Proto moved to https://github.com/ease-lab/vSwarm-proto/